src/app/app-content.jsx

```diff
@@ -145,6 +178,7 @@ const AppContent = observer(() => {
                     <BotStopped />
                     <TransactionDetailsModal />
                     <ToastContainer limit={3} draggable={false} />
+                    {shouldShowTradingAssessmentModal() && <TradingAssesmentModal />}
```

Comment:
@mayuran-deriv let's not add these logics in app-content. Let's create a component for Trading assessment and include all the logic there without cluttering the app-content. We will later refactor the app-content and make it cleaner.